poke.io.js

```diff
@@ -63,12 +62,6 @@ function getNeighbors(lat, lng) {
 function Pokeio() {
   var self = this;
   self.events = new EventEmitter();
-  self.j = request.jar();
```

Comment:
That is true, but i don't think we should worry about customising the jar object, as it is only used for logging in to PTC. If somebody needs to customise it, they could just replace the whole login function with their own.